Objective-C/Tests/QueryTest.m

```diff
@@ -2601,4 +2602,43 @@ - (void) testValueExpression {
     AssertEqualObjects([v asJSON], @21);
 }
 
+- (void) testUnaryQueryExpression {
+    NSDate* nw = [NSDate date];
+    CBLMutableDocument* doc1 = [self createDocument];
+    [doc1 setDate: nw forKey: @"now"];
+    [self saveDocument: doc1];
+    
+    CBLMutableDocument* doc2 = [self createDocument];
+    [self saveDocument: doc2];
+    
+    CBLUnaryExpression* notNull;
+    CBLUnaryExpression* notMiss;
```

Comment:
CBLUnaryExpression is an internal API. Is using notNullOrMissing enough to test it?

Reply:
Even though it is internal, we are not using the `not null` or `not missing` checks. For `isNotNullOrMissing`, we are using `negate: nullOrMissing`, which is also right but code is not getting validated for the result of NotNull and NotMissing. 

Impact of this test is not much in terms of exposed APIs, but more on an independent class basis. We are validating the CBLUnaryExpression for all possible usages.